src/quo2/components/tabs/account_selector.cljs

```diff
@@ -67,9 +67,10 @@
         label-text])
      [rn/view {:style (account-container-row background-color)}
       [rn/view {:style account-emoji-container}
-       [quo2/text account-emoji]]
+       [quo2/text {:style {:font-size 14}} account-emoji]]
```

Comment:
this is the wrong use of how we work with texts. Also you can nest styled texts so better to pass the element with this style from your component rather than update this component as this could effect all current uses of the tabs component

Reply:
ok cool, also let's avoid using any `status_im.*` namespaces as that is old code we don't want to use 👍

Reply:
@J-Son89 got it